MAUI/PDF-Viewer/Coordinate-Conversion.md

```diff
@@ -0,0 +1,119 @@
+
+You can obtain the PDF page coordinates using the `ConvertClientPointToPagePoint(clientPoint, pageNumber)` method by passing the client rectangle point and page number as input parameters. The following code sample explains how to convert a tapped client area position to a page point.
+
+{% tabs %}
+{% highlight c# %}
+//Subscribe the Tapped event of PDF Viewer
+pdfViewerControl.Tapped += PdfViewerControl_Tapped; 
+ 
+// Handle the tapped event.
+private void PdfViewerControl_Tapped(object sender, GestureEventArgs e)
+{
+    //Retrieve the tapped client area position.
+    Point clientPoint = e.Position;
+    //Retrieve the page number that corresponds to the client point.
+    int pageNumber = pdfViewerControl.GetPageNumberFromClientPoint(clientPoint)
+    //Retrieve the page point.
+    Point pagePoint = pdfViewerControl.ConvertClientPointToPagePoint(clientPoint, pageNumber);
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Convert PDF page coordinates to PDF Viewer’s coordinates
+
+You can obtain the PDF page coordinates using the `ConvertPagePointToClientPoint(pagePoint, pageNumber)` method by passing the page point and page number as input parameters. The following code sample explains how to convert a square annotation’s position in the page coordinates to a client point.
+
+{% tabs %}
+{% highlight c# %}
+//Get a shape annotation from the Annotations collection.
+SquareAnnotation squareAnnotation = pdfViewerControl.Annotations[0] as SquareAnnotation;
+//Get the annotation’s page number.
+int pageNumber = squareAnnotation.PageNumber;
+//Get the annotation bounds.
+Rectangle annotationBounds = squareAnnotation.Bounds;
+//Find the position of the annotation in page coordinates.
+Point pagePoint = new Point(annotationBounds.X, annotationBounds.Y);
+//Convert the page point to client point.
+Point clientPoint = pdfViewerControl.ConvertPagePointToClientPoint(pagePoint, pageNumber);
+{% endhighlight %}
+{% endtabs %}
+
+## Convert PDF page coordinates to scroll coordinates
+
+You can obtain the PDF Viewer’s scroll coordinates using the `ConvertPagePointToScrollPoint(pagePoint, pageNumber)` method by passing the page point and page number as input parameters. The following code example explains how to convert a square annotation’s position in the page coordinates to a scroll point.
+
+{% tabs %}
+{% highlight c# %}
+//Get a shape annotation from the Annotations collection.
+SquareAnnotation squareAnnotation = pdfViewerControl.Annotations[0] as SquareAnnotation;
+//Get the annotation bounds.
+Rectangle annotationBounds = squareAnnotation.Bounds;
+//Get the annotation’s page number.
+int pageNumber = squareAnnotation.PageNumber;
+//Find the position of the annotation in page coordinates.
+Point pagePoint = new Point(annotationBounds.X, annotationBounds.Y);
+//Convert the page point to scroll point.
+Point scrollPoint = pdfViewerControl.ConvertPagePointToScrollPoint(pagePoint, pageNumber);
+{% endhighlight %}
```

Comment:
<img width="352" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/2052ce9c-90cd-464e-812b-b831a759b975">

use the same values, spaces as in Xamarin. Text can also be improved using the same size, color and font used in Xamarin

Reply:
addressed